actions/selectedVariablesActions.js

```diff
@@ -28,3 +31,18 @@ export const setNumberOfBreaks = number => ({
   type: SET_NUMBER_OF_BREAKS,
   payload: number
 });
+
+export const setSelectedLocalMoranVariable = variable => ({
```

Comment:
@justin-kleid We can, in another PR, convert the js file to ts and add types to all the variables and function arguments